pyproject.toml

```diff
@@ -10,12 +10,6 @@ repository = "https://github.com/rocioar/flake8-django"
 classifiers=[
     "Framework :: Flake8",
     "License :: OSI Approved :: MIT License",
-    "Programming Language :: Python",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.5",
-    "Programming Language :: Python :: 3.6",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
```

Comment:
Do we still need to have the label for 3.8 and 3.7 since they will be supported?

Reply:
Poetry automatically adds whole range of supported Python version classifiers.  See [`cleo` pyproject](https://github.com/python-poetry/cleo/blob/1.0.0a5/pyproject.toml) and [`cleo` on PyPI](https://pypi.org/project/cleo/1.0.0a5/).

Reply:
nice!